src/YieldDistributor.sol

```diff
@@ -97,8 +101,12 @@ contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
      * @return uint256 The voting power of the user
      */
     function getCurrentVotingPower(address _account) public view returns (uint256) {
-        return
-            this.getVotingPowerForPeriod(BREAD, lastClaimedBlockNumber - cycleLength, lastClaimedBlockNumber, _account);
+        return this.getVotingPowerForPeriod(
```

Comment:
is the BB contract is not set for some reason, could revert.
could do following if issue is relevant:
```
uint256 _baseVotingPower = getCurrentVotingPower(_account);
uint256 _derivativeVotingPower = (address(BB) != address(0))
       ? this.getVotingPowerForPeriod(BB, lastClaimedBlockNumber - cycleLength, lastClaimedBlockNumber, _account)
       : 0;

return _baseVotingPower + _derivativeVotingPower;
```

Reply:
addressed with f909a84